bee-network/src/network/host.rs

```diff
@@ -51,7 +52,9 @@ pub mod integrated {
 
         async fn start(node: &mut N, config: Self::Config) -> Result<Self, Self::Error> {
             node.spawn::<Self, _, _>(|shutdown| async move {
-                network_host_processor(config, shutdown).await;
+                network_host_processor(config, shutdown)
+                    .await
+                    .expect("network host processor.");
```

Comment:
Maybe this `expect` should be a bit more descriptive?

Reply:
I agree. I personally see very little point in using `expect` tbh. If we're fine with this panicking in the first place, I'd just unwrap and have the actual fully fledged error instead of a generic message.

Reply:
yea, you got me here. It is something where I have not yet found a consistent and defendable way of doing this, but I feel like this enters "personal perference" territory. I've seen both actually in other codebases...very detailed `expect`s and very short ones. Regarding your 2nd sentence:
`expect` does print the fully fledged error additionally to the message passed to `expect`. That's exactly the reason why this is `expect` can be kept short. The error may be thrown in several places, so the message is just a hint for the programmer already providing some context to the error which might help in debugging scenarios.

Reply:
@pvdrz any reason from your side why the `expect` should be any more descriptive? Keeping them short is my current prefernce because the full error is printed alongside anyway. I cannot give much more information here apart from the "context" rather than the codeline and the filename only.

Reply:
hmm you're right. I've changed my opinion. A simple `unwrap` should be better if we get all the error information anyway.